Food/Places.md

```diff
@@ -0,0 +1,3 @@
+# Places to eat

```

Comment:
@sukhatri  Please ignore this message. I am just doing some search on git pull comment and how to do pull request comment part